### DISCUSSION TITLE
Isogeometric Analysis in MOOSE

### DISCUSSION THREAD

**Discussion by NorbertHofbauer** [idaholab/moose#21054] [Q&A Getting Started] [answered]
Hello,

where can i find the c-frame.step file for the IGA example?

https://mooseframework.inl.gov/modules/tensor_mechanics/examples/cframe_iga.html

best,
norbert

**Comment by GiudGiud** [comment, marked as answer]
Hello

It's there
/modules/tensor_mechanics/examples/cframe_iga/c-frame.step

Guillaume

**Reply by NorbertHofbauer** [reply]
Thanks!

i hadn't looked into the local files. i thought i could find the file somewhere in the online documentation.

wish you a nice weekend!
